docs/uglocalpv_volume_encrypt.md

```diff
@@ -0,0 +1,197 @@
+# How to use encryption with OpenEBS Local PV Hostpath volume
+
+## Introduction
+
+LUKS encrypted devices can be used to create OpenEBS Local PV Hostpath volumes by following the instructions. The following steps are mainly applicable for Local PV volumes running on **OpenEBS version 1.9 and above** only.
+
+## Instructions
+
+The following are the steps to provide encrypted OpenEBS Local PV volume capacity running on OpenEBS version 1.9 and above. Before you start make sure to add additional block devices to each node to be encrypted using LUKS.
+
+### Step 1: Find the new device name:
```

Comment:
Should we say Perform Step1 to Step 6 on each worker node where the disk is attached?

Reply:
Yes, this still needs some work
Missing parts:
- Key management
- Node restart
- Note to run the same on all nodes
- Complete cStor instructions